functions/env_vars/main.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+
+# [START functions_env_vars]
+def env_vars(request):
+    return os.environ['FOO']
```

Comment:
This will throw an exception if FOO isn't set. If that's the desired outcome, that's fine; otherwise perhaps use os.getenv with a default value of something like "environment variable not set"

Reply:
Hi @andrewsg ! The documentation has one step specified before using this snippet, instructing developers to set up the `FOO` env var; though I agree it would be better to add a default value. Will add a new commit.